BossMod/BossModule/ArenaBounds.cs

```diff
@@ -11,6 +10,9 @@ public abstract record class ArenaBounds(float Radius, float MapResolution)
     public RelSimplifiedComplexPolygon ShapeSimplified { get; private set; } = new();
     public List<RelTriangle> ShapeTriangulation { get; private set; } = [];
     private readonly PolygonClipper.Operand _clipOperand = new();
+    public static readonly Dictionary<object, object> StaticCache = [];
+    public readonly Dictionary<object, object> Cache = [];
+    public WPos Center;
```

Comment:
This is not appropriate here, the bounds were specifically refactored to *not* have the center (because in certain cases bounds are centered on player, and I don't want to rebuild them every frame).